src/Microsoft.AspNet.OData/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -225,6 +225,31 @@ public static Uri GetNextPageLink(this HttpRequestMessage request, int pageSize)
             return GetNextPageHelper.GetNextPageLink(requestUri, request.GetQueryNameValuePairs(), pageSize);
         }
```

Comment:
why not just call GetNextPageLink with null or omitted instance and objToSkipTokenValue params?